app/models/tutor.rb

```diff
@@ -48,7 +47,8 @@ class Tutor < ApplicationRecord
                            allow_blank: true
   validates :other_phone,  format: { with: VALID_PHONE_REGEX },
                            allow_blank: true
-  validates :smartt_id,    format: { with: VALID_SMARTT_REGEX }
+  validates :smartt_id,    format: { with: VALID_SMARTT_REGEX },
+                           allow_blank: true
```

Comment:
`smartt_id` is not known when the tutors or students are entered into the system. It's only given to the coordinators after the first monthly cycle of data entry into smart.